build/build-bundle.js

```diff
@@ -64,7 +64,14 @@ async function browserifyFile(entryPath, distPath) {
     .ignore('raven')
     .ignore('mkdirp')
     .ignore('rimraf')
-    .ignore('pako/lib/zlib/inflate.js');
+    .ignore('pako/lib/zlib/inflate.js')
+    .ignore('file') // required by jsonlint-mod
+    .ignore('system'); // required by jsonlint-mod
+
+  // There is no way to add './doug-json-parse' to ignored packages via public API
+  // w/o browserify resolving the path into an absolute path
```

Comment:
thanks for documenting these. They are pretty annoying to have to manually prune :(